src/main/java/seedu/address/model/util/Date.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.model.util;
```

Comment:
I think makes more sense to put Date in the transaction package (it's not really a util class, and we are going to use it as part of transactions)